tailwind.config.js

```diff
@@ -13,20 +14,25 @@ module.exports = {
         'light-wool': "url('/assets/light-wool.png')"
       })
     },
-    // colors: {
-    //   black: gruvbox.dark.hard,
-    //   white: colors.white,
-    //   paper: gruvbox.light[0],
-    //   primary: gruvbox.dark[0],
-    //   secondary: gruvbox.red.neutral,
-    //   ...gruvbox,
-    // },
+    colors: {
+      ...colors,
+      apnaGreen: "#04D98B",
+      dateGreen: "#038C5A",
+      apniHeading: "#014029",
+      neechuBG: "#3B3C40",
+      neechuText: "#f2f2f2",
+      ivory: "#FFFFF0",
+    },
```

Comment:
Please rename the color keys to more semantic names.

Reply:
https://github.com/ILUGD/planet/pull/17/commits/6d34aa5839f0d39394d8fc15a7ca7b6301b04337 - resolved here